webapp/src/components/propertyValueElement.tsx

```diff
@@ -115,6 +116,19 @@ const PropertyValueElement = (props:Props): JSX.Element => {
         )
     }
 
+    if (propertyTemplate.type === 'checkbox') {
+        return (
+            <Switch
+                isOn={Boolean(propertyValue)}
+                onChanged={(newBool) => {
+                    const newValue = newBool ? 'true' : '';
```

Comment:
There's probably a longer-term architecture question to think about here (fyi @jespino and @sbishel). 

It's possible to overload changePropertyValue to accept boolean values (like we do for arrays in the [multi-select PR](https://github.com/mattermost/focalboard/pull/415/files)). That would be cleaner, but I can't think of any problems with this approach either (which is simpler). One minor thing is maybe to store true as '1' (to make conversion to number easier), but that's 0/5.

As a side-note, we should probably test all the conversions between property types (i.e. by changing the type of an existing property with values, fyi @ogi-m) and flag any issues.

Reply:
I think I prefer to store the right value when it is possible, in this case, a boolean, is a boolean, moving into this "true"/"false" approach is something that is hard to change later.